addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -267,6 +269,26 @@ def _compute_can_approve(self):
             else:
                 allocation.can_approve = True
 
+    def _inverse_date_from(self):
+        for record in self:
+            if record.diplay_date_from:
+                date = record.display_date_from
+                record.date_from = datetime(date.year, date.month, date.day)
+            
+    def _compute_date_from(self):
```

Comment:
```suggestion
    @api.depends('date_from')
    def _compute_date_from(self):
```
You never know if another function modifies `date_from`, it should behave the same as before